server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/MailDispatcher.java

```diff
@@ -109,12 +101,12 @@ public MailDispatcher build() {
     private final boolean ignoreError;
     private final String errorProcessor;
 
-    private MailDispatcher(MailStore mailStore, MailetContext mailetContext, boolean consume, boolean ignoreError, String errorProcessor) {
+    private MailDispatcher(MailStore mailStore, MailetContext mailetContext, boolean consume, String errorProcessor) {
         this.mailStore = mailStore;
         this.consume = consume;
         this.mailetContext = mailetContext;
         this.errorProcessor = errorProcessor;
-        this.ignoreError = ignoreError;
+        this.ignoreError = errorProcessor.equalsIgnoreCase("ignore");
     }
```

Comment:
I think we can remove the ignore/propagate parsing. Our mailet pipeline already handles that. cf `ProcessorImpl`

Reply:
not sure if we need to customize the onMailetException handling though

Reply:
yes, it looks like a customized